assets/css/style.css

```diff
@@ -291,3 +308,62 @@ button:active {
 .about-myself p {
   margin-bottom: 80px;
 }
+
+.form-text {
+  margin-top: 50px;
+  width: 327px;
+  margin-left: 30px;
+  margin-bottom: 58px;
+}
+
+.input-email {
+  display: flex;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  width: 327px;
+  height: 48px;
+  margin-bottom: 24px;
+  margin-left: 20px;
+}
+
+.input-name {
+  display: flex;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  width: 327px;
+  height: 48px;
+  margin-bottom: 24px;
+  margin-left: 20px;
+}
+
+.full-name {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+  border: 0;
+  justify-content: center;
+  align-items: center;
+  padding: 12px;
+}
+
+.email {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+  border: 0;
+  justify-content: center;
+  align-items: center;
+  padding: 12px;
+}
```

Comment:
[OPTIONAL] Nice work implementing the features required for your form👍 Currently your email and username input currently doesn't have full width and I think it can be better if you gave them a width of 100%.
see the image for a better illustration🤗
![your_inputs](https://user-images.githubusercontent.com/64643995/212988989-76bf1bc5-4055-4e39-bc7f-1b49d6956a1c.png)